docs/remasters/15plus25/demo_debug.md

```diff
@@ -0,0 +1,99 @@
+# [Kingdom Hearts 2.5 HD Remix](remasters/15plus25/index.md) - KH2FM+ Debug Menu

```

Comment:
`(remasters/15plus25/index.md)` should be `(index.md)` . In the way you wrote it, the link would expect to go to https://openkh.dev/remasters/15plus25/remasters/15plus25/index.md. The reason is because the document it's already in `remasters/15plus25/`, so you do not need to specify the full path of `index.md`.

Reply:
Thank you, good catch.